recipes/date/all/conanfile.py

```diff
@@ -23,7 +31,46 @@ def source(self):
         for patch in self.conan_data["patches"][self.version]:
             tools.patch(**patch)
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["ENABLE_DATE_TESTING"] = False
+        if self.settings.os == "Windows":
+            cmake.definitions["USE_TZ_DB_IN_DOT"] = False
+            cmake.definitions["USE_SYSTEM_TZ_DB"] = False
+        else:
+            cmake.definitions["USE_TZ_DB_IN_DOT"] = self.options.use_tz_db_in_dot
+            cmake.definitions["USE_SYSTEM_TZ_DB"] = self.options.use_system_tz_db
+        cmake.configure(source_folder=self._source_subfolder)
+        return cmake
+
+    def requirements(self):
+        if self.settings.os == "Windows" or not self.options.use_system_tz_db:
+            self.requires("libcurl/7.67.0")
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
     def package(self):
-        self.copy(pattern="LICENSE.txt", dst="licenses", src=self._source_subfolder)
-        self.copy(pattern="date.h", dst=os.path.join("include", "date"),
-                  src=os.path.join(self._source_subfolder, "include", "date"))
+        self.copy(pattern="LICENSE.txt", dst="licenses",
+                  src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "CMake"))
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
+        if tools.os_info.is_linux:
```

Comment:
```suggestion
        if self.settings.os == "Linux"
```
(for cross-building)